### PR TITLE
Fix typos in the vignettes

### PR DESCRIPTION
**Pull request by mcol**
Corrects a number of typos I've found. ~~I don't think this requires an entry in the Changelog, but let me know if you'd like one.~~ Scratch that, I've noticed that in the past one was added, so there's a second commit for that.

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,14 @@
+2024-11-03  Marco Colombo <mar.colombo13@gmail.com>
```
Comment (eddelbuettel) [inline ChangeLog:1]: Two spaces between date and name (which you have) and name and email (one missing).

One empty line after the header.

**Comment by eddelbuettel** [comment]
Thanks for all that attention to detail, I will take a closer look later and likely merge.  Much appreciated!

File: ChangeLog (hunk 1 above)
Comment (mcol) [reply ChangeLog]: Done :smile:

**Review by eddelbuettel — APPROVED** [review]
Really appreciate the attention to detail here catching all these little errors.

**Pull request merged**